easybuild/easyconfigs/b/Bader/Bader-1.02-intel-2018a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'Bader'
+version = '1.02'
+
+homepage = 'http://theory.cm.utexas.edu/henkelman/code/bader/'
+description = "A fast algorithm for doing Bader's analysis on a charge density grid."
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['http://theory.cm.utexas.edu/henkelman/code/bader/download/v%(version)s']
+sources = [{
+    'filename': 'bader-%(version)s.tar.gz',
+    'download_filename': 'bader.tar.gz',
+}]
+checksums = ['9defc5d005521a3ed8bad7a3e2e557283962269eddf9994864c77b328c179c63']
+
+skipsteps = ['configure']
+
+buildopts = '-f makefile.lnx_ifort LINK=""'
```

Comment:
The `LINK=""` part looks weird, can you include a comment to explain it?

Reply:
the makefile uses `LINK` (instead of `LDFLAGS`) to add `-static`. But in this case it's pretty much irrelevant either way (dynamic or static)...

Reply:
OK, just clarify that with a comment above this line, it looks a bit fishy. :)